scss/stanford_media.admin.scss

```diff
@@ -56,3 +56,9 @@
     display: block;
   }
 }
+
+drupal-media {
+  &[data-align="right"] {
+    float: right;
```

Comment:
Float??? Are you sure? Floats open up a bag of alignment problems but make sense if we want text wrapping.

Reply:
That's what the `align-right` class does. And this helps the placement of the move icon on the media item. So a user can drag and drop the image to a different spot